examples/meta/generator/targets/csharp.json

```diff
@@ -101,14 +103,14 @@
         "Access": {
             "Vector": "$identifier[$indices]",
             "Matrix": "$identifier[$indices]",
-            "BoolVector": "$identifier.get_element($indices)",
-            "BoolMatrix": "$identifier.get_element($indices)"
+            "BoolVector": "$identifier[$indices] == 1 ? true : false",
+            "BoolMatrix": "$identifier[$indices] == 1 ? true : false"
```

Comment:
same as java, because boolean arrays are always a pain..

Reply:
Think the matrix meta example might be a blocker... curios

Reply:
No, it works fine :)